app/src/main/java/com/sdp/movemeet/backend/providers/AuthenticationInstanceProvider.java

```diff
@@ -16,6 +14,9 @@ private AuthenticationInstanceProvider () {
     }
 
     public static FirebaseAuth getAuthenticationInstance() {
-        return instance.fAuth;
+        if (fAuth == null) {
+            fAuth = FirebaseAuth.getInstance();
```

Comment:
We can get the instance if fAuth is null ?

Reply:
Yes, Kepler made this modification here with a commit to my branch. I guess, the purpose is precisely to set `fAuth` as an instance of Firebase Authentication if it is still `null`.